src/modules/launcher/Wox.Test/Plugins/FolderPluginTest.cs

```diff
@@ -10,6 +10,7 @@
 
 namespace Wox.Test.Plugins
 {
+#pragma warning disable CA1812 // Suppressing warning for uninstantiated internal class
```

Comment:
I think we should place this error in a global suppression file for better code readability. You can get more info here: https://docs.microsoft.com/en-us/visualstudio/code-quality/in-source-suppression-overview?view=vs-2019#global-level-suppressions

Reply:
Thanks @somil55. I've updated the PR and added the suppression in GlobalSuppressions.cs instead of inline.